Source/Portal/HelperHeaders/amsuGetHelper.h

```diff
@@ -0,0 +1,14 @@
+﻿#pragma once
+
+#include "CoreMinimal.h"
+
+// TODO upgrade and refactor - use templates 
+inline APlayerController* GetPlayerController(AActor* Actor)
```

Comment:
```suggestion
APlayerController* GetPlayerController(AActor* Actor)
```
declaration in .h
definition in .cpp